Tabs.js

```diff
@@ -0,0 +1,78 @@
+class TabLink {
+  constructor(element) {
+    // Assign this.element to the passed in DOM element
+    this.element = element;
+
+    // Get the custom data attribute on the Link
+    this.data = this.element.dataset.tab;
+
+    // Using the custom data attribute get the associated Item element
+    this.itemElement = document.querySelector(
+      `.tabs-item[data-tab="${this.data}"]`
+    );
+
+    // Using the Item element, create a new instance of the TabItem class
+    this.tabItem = new TabItem(this.itemElement);
+
+    // Add a click event listener on this instance, calling the select method on click
+    this.element.addEventListener("click", this.select.bind(this));
+  }
+
+  select() {
+    // Get all of the elements with the tabs-link class
+    const links = document.querySelectorAll(".tabs-link");
+
+    // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
+    //Array.from(links).forEach(() => {
+    //this.element.classList.remove('tabs-link-selected');
+    //})
```

Comment:
It's worth noting that because a `querySelector` method was used to grab the `links`, there's no need to convert it to an array with `Array.from()` - the `.forEach` method is included by default.